### PR TITLE
ENH:DataProbe:  splitting processEvent method for override

### PR DESCRIPTION
**Pull request by Punzo**
As discussed in ENH: units/coordinates display in DataProbe #309
I need to override part of the method processEvent in DataProbe. 
I'd like to propose to split the method eventProcess so I can override only 
what is needed.

@pieper , @jcfr  , @lassoan. may you take a fast look at it? Thank you.

### CODE REVIEW

**Comment by lassoan** [comment]
Looks good except GetInfo is a very bad method name. Please rename it to something that explains what the method actually does.

**Comment by Punzo** [comment]
GetCursorPositionInfo?

**Comment by lassoan** [comment]
Much better. Maybe UpdateCursorPositionDisplayText would be even more accurate.

**Comment by pieper** [comment]
Agreed - I like UpdateCursorPositionDisplayText

**Comment by Punzo** [comment]
thanks for the feedback, I changed the name,

Davide.

File: Modules/Scripted/DataProbe/DataProbe.py [hunk 1]
```diff
@@ -331,6 +294,52 @@ def _roundInt(value):
...
+              spacing=spacing
+              )
+
+    def _roundInt(value):
+      try:
+        return int(round(value))
+      except ValueError:
+        return 0
+
+    hasVolume = True
```
Comment (jcfr) [inline Modules/Scripted/DataProbe/DataProbe.py]: @Punzo  I am integrating the changes. Note that I will keep this as `hasVolume = False` to match the existing behaviour.

**Comment by jcfr** [comment]
Closing. Superseded by #338

**Pull request closed without merging**